wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -371,6 +377,12 @@ impl super::TermWindow {
                 }
                 TabBarItem::None => {}
             },
+            WMEK::Move => match item {
+                TabBarItem::None => {
+                    context.set_window_drag_position(event.screen_coords);
+                }
+                _ => {}
+            },
```

Comment:
If we pass down the `item` parameter from `mouse_event_ui_item`, we could assign the drag region based on the `x`, `y`, `width`, and `height` 

Stylistic suggestion for this bit is to use `matches!`:

```suggestion
            WMEK::Move => if matches!(item, TabBarItem::None) {
                    context.set_window_drag_position(event.screen_coords);
            },
```

Reply:
The formatter breaks it into:

```rust
            WMEK::Move => {
                if matches!(item, TabBarItem::None) {
                    context.set_window_drag_position(event.screen_coords);
                }
            }
``` 

So I guess it's not much of an improvement